.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "airbnb-base",
+    "overrides": [
+    ],
```

Comment:
You can remove this as there are no overrides to include, same goes for the `rules` block below